pkg/deploy/api/types.go

```diff
@@ -311,6 +311,10 @@ type DeploymentConfigSpec struct {
 	// or failing. Post strategy hooks and After actions can be used to integrate successful deployment with an action.
 	Test bool
 
+	// Paused indicates that the deployment config is paused resulting in no new deployments on template
+	// changes or changes in the template caused by other triggers.
```

Comment:
I think the command will succeed but nothing will be deployed until the config is un-paused.